W-16350651-Flow-count-in-Studio-CP (#583)
mulesoft/docs-studio: modules/ROOT/pages/flow-counting-studio.adoc

= Tracking the Number of Billable Flows in Studio
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> In Anypoint Studio 7.20 and later, you can track the number of billable flows your project has.

Reviewer: "flows in your project"?

Author: Yes! Thanks!

include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-intro"]

include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-billable-flows"]

== Before You Begin

include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-prereq"]

> == Counting the Flows of Your Project in Studio

Reviewer: ```suggestion
== Count the Flows in Your Project using Studio
```

Reviewer: Imperative for tasks

Author: OK! Thanks!

. Select your project.

> . Click the *Flow Count* icon (image:flow-count-icon.png[2,2%""]) in the task bar.

Reviewer: You might need a comma after the %
Should there be another % for the first 2?

Reviewer: I think it's "taskbar".

Author: In my local build the icon looks fine. I'll make the change anyway, just in case 😄 
Thanks!

Reviewer: It's just that I'm not sure if "2" is the same as "2%". And usually, there's a comma between those attributes.

> . Click the *Flow Count* icon (image:flow-count-icon.png[2,2%""]) in the task bar.

Reviewer: ```suggestion
. Click *Flow Count* (image:flow-count-icon.png[2,2%""]) in the taskbar.
```

Reviewer: I think it's taskbar, but our docs have more instances of "task bar" so <shrug>

Author: It's "taskbar". Great catch! Thanks!

+

> Studio displays the number if billable flows your Anypoint Platform Organization is charged for running deployed flows.

Reviewer 2: number of*

Author: Grat catch! Thanks!

> Studio displays the number of billable flows your Anypoint Platform Organization is charged for running deployed flows.

Reviewer: ```suggestion
Studio displays the number of billable flows your Anypoint Platform organization is charged for running deployed flows.
```

Author: OK! Thanks!

+

> image:flow-count.png["A table displays the billable flow names and type."]

Reviewer: ```suggestion
image:flow-count.png["A table showing the billable flow names and type"]
```

Author: OK! Thanks!
